administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-17.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE `#__guidedtour_steps`
+SET `ordering` = `ordering` + 1
+WHERE `title` = 'COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_STEP_SAVECLOSE_TITLE';
+
+UPDATE `#__guidedtour_steps`
+SET `ordering` = `ordering` + 1
+WHERE `title` = 'COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_STEP_CONGRATULATIONS_TITLE';
```

Comment:
```suggestion
WHERE `title` IN ('COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_STEP_SAVECLOSE_TITLE', 'COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_STEP_CONGRATULATIONS_TITLE');
```
Why 2 statements where 1 is enough?

Reply:
P.S.: The `ordering` = `ordering` + 1 works for each record individually if you update multiple records at once.